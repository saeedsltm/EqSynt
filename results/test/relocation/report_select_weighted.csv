OT,Lat,LatErr,Lon,LonErr,Dep,DepErr,NST,RMS,GAP
2020-01-01 00:01:40.200000,35.967,  1.90,51.792,  4.40, 22.6,  4.20,  8, 0.3,224
2020-01-01 00:03:19.200000,36.857, 19.20,50.891, 73.60, 18.1,109.30,  5, 0.2,320
2020-01-01 00:05:00.200000,35.552, 22.00,50.984, 42.10, 25.4,999.90,  3, 0.4,314
2020-01-01 00:06:40.000000,35.357, 20.10,52.227, 13.50, 10.0, 20.70,  6, 0.3,192
2020-01-01 00:08:20.500000,35.634, 59.80,51.596, 51.70,  2.8, 60.40,  6, 0.4,323
2020-01-01 00:10:00.300000,37.143,  4.70,52.275,  3.50, 19.8,  4.80,  7, 0.4,293
2020-01-01 00:11:39.200000,35.387,  8.30,51.863, 34.50, 37.5, 19.80,  4, 0.3,324
2020-01-01 00:13:20.000000,35.719,  1.40,52.284,  4.10,  4.0,  4.20,  7, 0.3,154
2020-01-01 00:15:00.300000,36.596,  5.70,51.977,  9.40, 16.7,  6.60,  6, 0.4,253
2020-01-01 00:16:40.300000,35.995,  1.90,51.735, 17.80,  7.0,  9.50,  6, 0.4,249
