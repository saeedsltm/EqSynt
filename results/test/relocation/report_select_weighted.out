 Year Date HRMM  Sec Latitud Error Longitud Error Depth Error NST  RMS GAP
 2020  1 1 0001 40.2  35.967   1.9   51.792   4.4  22.6   4.2   8  0.3 224
 2020  1 1 0003 19.2  36.857  19.2   50.891  73.6  18.1 109.3   5  0.2 320
 2020  1 1 0005  0.2  35.552  22.0   50.984  42.1  25.4 999.9   3  0.4 314
 2020  1 1 0006 40.0  35.357  20.1   52.227  13.5  10.0  20.7   6  0.3 192
 2020  1 1 0008 20.5  35.634  59.8   51.596  51.7   2.8  60.4   6  0.4 323
 2020  1 1 0010  0.3  37.143   4.7   52.275   3.5  19.8   4.8   7  0.4 293
 2020  1 1 0011 39.2  35.387   8.3   51.863  34.5  37.5  19.8   4  0.3 324
 2020  1 1 0013 20.0  35.719   1.4   52.284   4.1   4.0   4.2   7  0.3 154
 2020  1 1 0015  0.3  36.596   5.7   51.977   9.4  16.7   6.6   6  0.4 253
 2020  1 1 0016 40.3  35.995   1.9   51.735  17.8   7.0   9.5   6  0.4 249
